### DISCUSSION TITLE
how to send special mail tags?

### DISCUSSION THREAD

**Discussion by dfstudioit** [kennym/cf7-to-api#41] [Q&A]
hello, this plugin is very helpful.
[{ "cognome" : "[cognome]", "nome" : "[nome]", "telefono_fax" : "[cellulare]", "nota" : "[caf][_remote_ip]"}]

This is the json template i send to an address, and this is what i see in debug

[{ "cognome" : "xxxxx", "nome" : "xxxxxx", "telefono_fax" : "0123456789", "nota" : "caf001"}]

Why i don't see remote ip conversion?

**Comment by UlisesArcidiacono** [comment]
Hello @dfstudioit @kennym,
Have you found a workaround for this?

I am facing the same issue with **post related fields (_post_id and _post_url)** https://contactform7.com/special-mail-tags/#post-related
I need to include the _post_id and _post_url as part of the json POST.

Below my json. Everything works except those _post* fields.

{
“name”: “[your-name]”,
“email”: “[your-email]”,
“mensaje”: “[your-message]”,
“ref”: “[_post_id]”,
“url”: “[_post_url]”,
“tel”: “[tel-179]”,
“mudanza”: “[menu-823]”
}

![error-post-fields](https://user-images.githubusercontent.com/1843430/121907191-36537580-cd02-11eb-8dc9-2b197aa52db9.jpg)
